lib/Service/Sync/ImapToDbSynchronizer.php

```diff
@@ -109,6 +109,10 @@ public function syncAccount(Account $account,
 		$snoozeMailboxId = $account->getMailAccount()->getSnoozeMailboxId();
 		$sentMailboxId = $account->getMailAccount()->getSentMailboxId();
 		$trashRetentionDays = $account->getMailAccount()->getTrashRetentionDays();
+		// construct imap client and perform initial connection
+		$client = $this->clientFactory->getClient($account);
+		$client->login();
+		// perform mail sync on all mail boxes
```

Comment:
```suggestion
```

https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/#h2-b7593c698a940